src/main/java/scratch/UCERF3/inversion/UCERF3InversionConfiguration.java

```diff
@@ -260,7 +260,7 @@ public static UCERF3InversionConfiguration forModel(InversionModels model, Fault
 		double parkfieldConstraintWt = 1000;
 		
 		// get MFD constraints
-		List<MFD_InversionConstraint> mfdConstraints = targetMFDs.getMFD_Constraints();
+		List<MFD_InversionConstraint> mfdConstraints = (List<MFD_InversionConstraint>) targetMFDs.getMFD_Constraints();

```

Comment:
any reason not to make the method a `List<? extends MFD_InversionConstraint>`?

Reply:
I don't think there was a special reason for it